src/components/tl/mlx5/mcast/tl_mlx5_mcast_helper.c

```diff
@@ -393,6 +393,158 @@ ucc_status_t ucc_tl_mlx5_mcast_setup_qps(ucc_tl_mlx5_mcast_coll_context_t *ctx,
     return UCC_OK;
 }
 
+ucc_status_t ucc_tl_mlx5_mcast_create_rc_qps(ucc_tl_mlx5_mcast_coll_context_t *ctx,
```

Comment:
wdyt about reusing/refactoring
https://github.com/openucx/ucc/blob/b62b4723a7183496efc3792d90e7bfefcfd6d7ed/src/components/tl/mlx5/tl_mlx5_ib.c#L346
here?

Reply:
I would rather not inject verb objects like ibv_qp_init_attr into tl_mlx5_ib.c. We would like to keep tl_mlx5_ib.c all about mlx5dv primitives. mcast lib is currently only based on libverb.